src/client-apps/nuxeo-drive/nuxeo-drive-metrics-documentation.md

```diff
@@ -0,0 +1,192 @@
+
+## `NX-metric-drive-global`
+
+This header is sent with each request to the platform, and  contains generic information about the Drive installation, configuration and the Operating System.
+
+### Available metrics
+
+#### `installation.- type`
+
+Description: TBD
+- Example: `null`
+- type: `str`
+
+#### `execution.profile`
+
+Description: TBD
+- type: `str`
+- Example: `null`
+
+#### `execution.session.uid`
+
+Description: TBD
+- type: `str`
+- Example: `null`
+
+#### `metrics.custom`
+
+Description: TBD
+- type: `int`
+- Example: `null`
+
+#### `metrics.ga`
+
+Description: TBD
+- type: `int`
+- Example: `null`
+
+#### `metrics.sentry`
+
+Description: TBD
+- type: `int`
+- Example: `null`
+
+#### `updater.channel`
+
+Description: TBD
+- type: `str`
+- Example: `null`
+
+#### `execution.locale`
+
+Description: TBD
+- type: `str`
+- Example: `null`
+
+#### `os.machine`
+
+Description: TBD
+- type: `str`
+- Example: `null`
+
+#### `os.locale`
+
+Description: TBD
+- type: `str`
+- Example: `null`
+
+## `NX-metric-drive-request`
+
+This header is sent with each request to the platform, and contains usage and behaviour information about Nuxeo Drive features.
+
+### Direct Edit metrics
+
+#### `directEdit.conflict.hit`
+
+Description: TBD
+- type: `int`
+- Example: `null`
+
+#### `directEdit.recovery.hit`
+
+Description: TBD
+- type: `int`
+- Example: `null`
+
+#### `directEdit.error.count`
+
+Description: TBD
+- type: `int`
+- Example: `null`
+
+#### `directEdit.save.count`
+
+Description: TBD
+- type: `int`
+- Example: `null`
+
+
+### Direct Transfer metrics
+
+#### `directTransfer.session.number`
+
+Description: TBD
+- type: `int`
+- Example: `null`
+
+#### `directTransfer.duplicate.file.behavior`
+
+Description: TBD
+- type: `str`
+- Example: `null`
+
+#### `directTransfer.option.newFolder`
+
+Description: TBD
+- type: `int`
+- Example: `null`
+
+#### `directTransfer.session.status`
+
+Description: TBD
+- type: `str`
+- Example: `null`
+
+Synchronization metrics:
+
+#### `sync.error.label`
+
+Description: TBD
+- type: `str`
+- Example: `null`
+
+#### `sync.action`
+
+Description: TBD
+- type: `int`
+- Example: `null`
+
+#### `sync.time`
+
+Description: TBD
+- type: `int`
+- Example: `null`
+
+
+### Other metrics
+
+#### `app.crashed.hit`
+
+Description: TBD
+- type: `int`
+- Example: `null`
+
+#### `app.crashed.trace`
+
+Description: TBD
+- type: `str`
+- Example: `null`
+
+#### `filters.syncRoot.count`
+
+Description: TBD
+- type: `int`
+- Example: `null`
+
+# Nuxeo Drive asynchronous metrics
+
+Some of the metrics are calculated asynchronously over a period of time and thus cannot be sent directly to the server.</br>
+These metrics are sent to the `/site/api/v1/me` endpoint once calculated.
+To avoid spamming the endpoint too much and causing performance issues, Nuxeo Drive sends these metrics in a batch every 15 minutes (not configurable).</br>
```

Comment:
We added the "private" `custom_metrics_poll_interval` option (in seconds). Maybe should we talk about it here?